### ISSUE TITLE
Not a valid object name master

### ISSUE THREAD

**Issue by azu** [packsaddle/ruby-saddler-reporter-support-git#15]
https://travis-ci.org/jser/jser.github.io/builds/66056183#L362

I try to use [Saddler](https://github.com/packsaddle/ruby-saddler) and see this error:

```
/home/travis/.rvm/gems/ruby-2.2.0/gems/git-1.2.9.1/lib/git/lib.rb:917:in `command': git '--git-dir=/home/travis/build/jser/jser.github.io/.git' '--work-tree=/home/travis/build/jser/jser.github.io' cat-file '-t' 'master'  2>&1:fatal: Not a valid object name master (Git::GitExecuteError)
```
- https://github.com/packsaddle/ruby-saddler-reporter-support-git/blob/e670201840a507fb0006bcf863160b98cac1f59c/lib/saddler/reporter/support/git/repository.rb#L53

I think that this error is occured when **default branch** is not master.

![2015-07-26_21-33-44](https://cloud.githubusercontent.com/assets/19714/8893905/13609184-33de-11e5-9f62-f7f0c4e8d1c1.jpg)

The reproduce script is [here](https://github.com/jser/jser.github.io/blob/develop/test/travis-spellcheck.sh), but it was disabled.

**Comment by sanemat** [comment]
@azu Thank you for your reporting!

**Comment by sanemat** [comment]
We shipped `v0.1.8`. Thanks!

**Comment by azu** [comment]
Thanks for quick fix :+1:
